mail_activity_to_crm_request/models/mail_activity.py

```diff
@@ -0,0 +1,103 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from collections import defaultdict
+from datetime import date, datetime
+from dateutil.relativedelta import relativedelta
+import logging
+import pytz
+import random
+
+from odoo import api, exceptions, fields, models, _
+from odoo.osv import expression
+
+from odoo.tools.misc import clean_context
+from odoo.addons.base.models.ir_model import MODULE_UNINSTALL_FLAG
+
+_logger = logging.getLogger(__name__)
+
+class MailActivity(models.Model):
+    _inherit = 'mail.activity'
+
+    convert_activity_to_sup_req = fields.Boolean(
```

Comment:
I would suggest this name instead:
```suggestion
    create_support_request = fields.Boolean(
```